python/ray/exceptions.py

```diff
@@ -326,6 +326,22 @@ def __str__(self):
         )
 
 
+class OutOfDiskError(RayError):
+    """Indicates that the local disk is full.
+
+    This is raised if the attempt to store the object fails
+    because both the object store and disk are full.
+    """
+
+    def __str__(self):
+        return super(OutOfDiskError, self).__str__() + (
+            "\n"
+            "The local object store is full and local disk is also full."
```

Comment:
Turns out the plumbing the exact disk size is a bit challenging. Let's defer that to another PR.

Reply:
How about at least listing the disk percentage set in the ray config?

Also, suggest: "The object cannot be created because the local object store is full and at least X% of the local disk is in use."